lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -99,3 +102,135 @@ def calibration_results(self, fc, diffusion_constant):
+        num_windows : int, optional
+            Number of windows to average for the uncalibrated force. Using a larger number of
+            windows potentially increases the bleed, but may be useful when the SNR is low.
+        """
+        super().__init__(bead_diameter)
+        self.driving_frequency_guess = driving_frequency_guess
+        self.sample_rate = sample_rate
+        self.viscosity = viscosity
+        self.temperature = temperature
+        self.num_windows = num_windows
+
+        # Estimate driving input and response
+        amplitude_um, self.driving_frequency = estimate_driving_input_parameters(
+            sample_rate, driving_data, driving_frequency_guess
+        )
+        self.driving_amplitude = amplitude_um * 1e-6
+        self.response_power_density, self.frequency_resolution = response_power_density(
+            force_voltage_data, sample_rate, self.driving_frequency, num_windows
+        )
+
+    def calibration_parameters(self):
+        return {
+            "Bead diameter": CalibrationParameter("Bead diameter", self.bead_diameter, "um"),
+            "Driving frequency (guess)": CalibrationParameter(
+                "Driving frequency (guess)", self.driving_frequency_guess, "Hz"
+            ),
+            "Sample rate": CalibrationParameter("Sample rate", self.sample_rate, "Hz"),
+            "Temperature": CalibrationParameter("Liquid temperature", self.temperature, "C"),
+            "Viscosity": CalibrationParameter("Liquid viscosity", self.viscosity, "Pa*s"),
+            "num_windows": CalibrationParameter("Number of averaged windows", self.num_windows, ""),
+        }
+
+    def _power_theory(self, f_corner):
+        """Compute the power spectral density expected for a given driving input."""
+        return self.driving_amplitude ** 2 / (2 * (1 + (f_corner / self.driving_frequency) ** 2))
+
+    def calibration_results(self, fc, diffusion_constant_volts, f_diode, alpha):
+        """Compute calibration parameters from cutoff frequency and diffusion constant.
+
+        Parameters
+        ----------
+        fc : float
+            Corner frequency, in Hz.
+        diffusion_constant_volts : float
+            Diffusion constant, in V^2/s
+        f_diode : float
+            Diode frequency.
+        alpha : Alpha
+            Fraction of PSD signal that is instantaneous
+        """
+        reference_peak = self(self.driving_frequency, fc, diffusion_constant_volts, f_diode, alpha)
+
+        power_exp = (self.response_power_density - reference_peak) * self.frequency_resolution
+        distance_response = np.sqrt(self._power_theory(fc) / power_exp)  # m/V
+
+        omega = 2.0 * np.pi * fc
+        temperature_kelvin = scipy.constants.convert_temperature(self.temperature, "C", "K")
+        k_temperature = scipy.constants.k * temperature_kelvin
+        gamma_0 = k_temperature / (distance_response ** 2 * diffusion_constant_volts)  # kg/s^2
+        kappa = omega * gamma_0  # N/m
```

Comment:
I think `omega` is superfluous here; considering it's only used in the assignment for `kappa` which left me searching through equations to figure out what special meaning this variable has. 

I think it'd be better to just have `kappa = 2.0 * np.pi * fc * gamma_0`. This also keeps it consistent with the definition in `PassiveCalibrationModel.calibration_results()`

Reply:
Yeah, before I simplified stuff there were some longer equations and omega was used in several places (to keep line lengths shorter). It is indeed not needed now. Removed. :)